src/platform-implementation-js/style/gmail.css

```diff
@@ -1240,7 +1233,6 @@ isn't visible behind it. */
   border: none;
   display: inline-flex;
   justify-content: center;
-  outline: none;
```

Comment:
Why was this removed?

Reply:
@dmy7r0 It looks like `outline` in this ruleset was just doubled up. The subsequent [one is here](https://github.com/InboxSDK/InboxSDK/pull/912/files#diff-4deb24efbdddeae857792fd63a7c2937123ae6c6f6863bd2bfbe6ccacdeaad88R1247)